### PR TITLE
CLOUDP-161428: Remove "Get" from getters

### PR DESCRIPTION
**Pull request by andmatei**
## Description
According the [Effective Go](https://go.dev/doc/effective_go#Getters) doc, getters should not contain "Get".
First commit changes the templates, second commit regenerates the client.

### CODE REVIEW

File: tools/config/go-templates/model_simple.mustache [hunk 1]
```diff
@@ -90,14 +90,16 @@ func New{{classname}}WithDefaults() *{{classname}} {
 
 {{#vars}}
 {{#required}}
-// Get{{name}} returns the {{name}} field value
+{{! X-XGEN-MODIFIED }}
```
Comment (andreaangiolillo) [inline tools/config/go-templates/model_simple.mustache:93]: What is this for?

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: tools/config/go-templates/model_simple.mustache (hunk 1 above)
Comment (andmatei) [reply tools/config/go-templates/model_simple.mustache:93]: Just to keep track of what we have modified in the templates, so we can easily merge the templates from upstream with our modifications.

**Comment by wtrocki** [comment]
Actually. This seems like it's not going to work. It breaks linting (name will overlap). Let's close and refine our strategy in what methods we want in model.

**Pull request closed without merging**